referrer-policy/no-referrer-policy/no-referrer-policy.subresource.py

```diff
@@ -0,0 +1,16 @@
+import os, json
+
+def main(request, response):
+    script_directory = os.path.dirname(os.path.abspath(__file__))
+    template_basename = "no-referrer-policy.subresource.template.html"
```

Comment:
I was thinking the same thing as I was repushing. Actually I think one python script should be sufficient for the whole referrer policy. As this whole thing is a test in itself, such refactoring should come easy and without fear of breaking stuff.